File: est datos/python/insercion.py
if __name__ == "__main__":
    arr = [34,6,123,54,12]
    print(arr)
    
    print("Ingrese un numero a insertar")
    num = int(input())
    print("Ingrese la posicion donde desea insertarlo (1 a 5)")
    pos = int(input())

    for i in range(4, pos-1, -1):
        arr[i] = arr[i-1]
    arr[pos-1] = num

    print("Arreglo despues de la insercion:")
    print(arr)

File: est datos/Js/insercion.js
let arr = [123,45,67,89,43,12];

alert("El arreglo inicial es: " + arr);

let dato = parseInt(prompt("ingrese el numero a insertar: "));

let pos = parseInt(prompt("ingrese la posicion donde desea insertar el numero: "));

for (let i = arr.length - 1; i>=pos ; i--){
    arr[i] = arr[i-1];
    
}
arr[pos - 1] = dato;

alert("El nuevo arreglo es: " + arr);
